specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ManagedEnvironments.json

```diff
@@ -841,6 +856,63 @@
           "x-ms-secret": true
         }
       }
+    },
+    "WorkloadProfile": {
+      "description": "Workload profile to scope container app execution.",
```

Comment:
>. [](http://example.com/codeflow?start=71&length=1)

None of the other descriptions end with "."

Reply:
I think it is inconsistent. :)
Refer 
- line 23, 24, 42, 63, 64 in ManagedEnvironments.json
- line 439, 453, 458 in ContainerApps.json